internal/github/github.go

```diff
@@ -51,30 +54,30 @@ func (g *GitHub) getToken() error {
 
 	key, err := ioutil.ReadFile(g.keyFile)
 	if err != nil {
-		return fmt.Errorf("getToken: could not read private key: %v", err)
+		return errors.Wrap(err, "could not read private key")
 	}
 
-	block, _ := pem.Decode(key)
-	if block == nil {
-		return fmt.Errorf("getToken: could not decode pem private key: %v", err)
+	block, err := pem.Decode(key)
```

Comment:
@bradleyfalzon I dont know why this error was ignored, but if there was a good reason, let me know